crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -261,134 +249,6 @@ pub(crate) async fn farm_multi_disk(
     anyhow::Ok(())
 }
 
-async fn configure_dsn(
```

Comment:
Looks like this was moved and a lot of other changes were introduced at the same time in the same commit. I'm stopping reviewing for now on commit 1694a31108b2c17ceb393117bbc0d0ae2e344633.